### PR TITLE
Bugfix: show courses in all departments + small screens

### PR DESCRIPTION
**Pull request by RedTo**
Issues: #236 + #237 

image
- There are now courses in every department (at least in those which have courses on nine).
- Course info fits screen
- Course list has selected the user department by default.
- Course list has default department '13' if user have not specified a department on nine.

### CODE REVIEW

File: lib/widgets/courseList.dart [hunk 1]
```diff
@@ -40,7 +40,7 @@ class CourseListState extends State<CourseList> {
   final TextEditingController c1 = new TextEditingController();
   final bool shouldFilterByFavorites;
   bool shouldSearch = false;
-  String filter = "09";
+  String filter = "13";
   String searchValue = "";
```
Comment (AndiAndreas) [inline lib/widgets/courseList.dart]: Ist das richtig dass du standardmäßig nach fk13 suchen willst. Wir erhalten doch nur von fk09 sachen

Comment (AndiAndreas) [reply lib/widgets/courseList.dart]: Nach Screenshot erhalten wir mittlerweile auch andere departments oder?

Comment (RedTo) [reply lib/widgets/courseList.dart]: That's the reason, why this pull request exists ;-)

Comment (RedTo) [reply lib/widgets/courseList.dart]: On nine CiE is placed under department 13, for this reason I though it might be good to use it as a default.

**Pull request merged**